feat(changelog): transactional-email-added-xml-attachments-are-now-avail 2024-04-19 (#3087)
scaleway/docs-content: changelog/april2024/2024-04-19-transactional-email-added-xml-attachments-are-now-avail.mdx

---
title: XML attachments are now available!
status: added
author:
    fullname: 'Join the #transactional-email channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-04-19
category: managed-services
product: transactional-email
---

> You can now send XML as attachments in your transactional emails. The supported MIME are the following:

Reviewer: ```suggestion
You can now send XML attachments in your transactional emails. The following MIME types are supported:
```

- `application/xml`
- `text/xml`

> Find out about all the [attachment types available](https://www.scaleway.com/en/docs/managed-services/transactional-email/reference-content/tem-capabilities-and-limits/).

Reviewer: ```suggestion
Find out about the full range of available attachment types in our [documentation](https://www.scaleway.com/en/docs/managed-services/transactional-email/reference-content/tem-capabilities-and-limits/).
```

---
title: XML attachments are now available!
status: added
author:
    fullname: 'Join the #transactional-email channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-04-19
category: managed-services
product: transactional-email
---

You can now send XML as attachments in your transactional emails. The supported MIME are the following:
- `application/xml`
- `text/xml`

> Find out about all the [attachment types available](
> https://www.scaleway.com/en/docs/managed-services/transactional-email/reference-content/tem-capabilities-and-limits/).

Reviewer 2: ```suggestion
Find out about all the [attachment types available](https://www.scaleway.com/en/docs/managed-services/transactional-email/reference-content/tem-capabilities-and-limits/).
```
